src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -28,6 +28,10 @@
     <NuGetConfigFile Condition="'$(NuGetConfigFile)' == '' and Exists('$(ProjectDirectory)NuGet.config')">$(ProjectDirectory)NuGet.config</NuGetConfigFile>
     <NuGetConfigFile Condition="'$(NuGetConfigFile)' == '' and Exists('$(ProjectDirectory)NuGet.Config')">$(ProjectDirectory)NuGet.Config</NuGetConfigFile>
 
+    <OriginalNuGetConfigFile>$(NuGetConfigFile)</OriginalNuGetConfigFile>
+    <!-- Update nuget.config property to point to modified file that will be created/updated during build. -->
+    <NuGetConfigFile Condition="'$(OriginalNuGetConfigFile)' != ''">$(BaseIntermediateOutputPath)$([System.IO.Path]::GetFileName('$(OriginalNuGetConfigFile)'))</NuGetConfigFile>
```

Comment:
Double check that `BaseIntermediateOutputPath` is already defined at this point in the evaluation.

Reply:
Already verified changes locally before pushing this PR. It is defined in `Directory.Build.props` in VMR root.